src/Modal/styles.css

```diff
@@ -0,0 +1,35 @@
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
```

Comment:
```suggestion
  left: -50%;
```

Super cool bounce animation that looks like in-game part 1 😎